docs/how-tos/move-robot-arm.md

```diff
@@ -0,0 +1,211 @@
+{{< /alert >}}
+
+## Access the arm
+
+{{< table >}}
+{{% tablestep link="/configure/" %}}
+**1. Configure a machine with an arm**
+
+{{% snippet "setup.md" %}}
+
+[Configure an arm](/components/arm/#configuration).
+
+{{<imgproc src="/how-tos/access-arm/config.png" resize="500x" class="fill aligncenter" style="width: 400px" declaredimensions=true alt="Configuration builder UI with a blank arm component">}}
+
+Refer to the [documentation for the model](/components/arm/#configuration) for information about your arm's configuration attributes.
+Save your config.
+
+{{% /tablestep %}}
+{{% tablestep %}}
+**2. Get the position of the end effector**
+
+One way to describe the state of a robot arm is with the position of the end effector, or the "tool" or "hand" of the arm.
+The code to get this information is a part of the autogenerated code sample the Viam app provides for your machine.
+Go to the **Code sample** page of the **CONNECT** tab and select either Python or Go.
+
+{{% snippet "show-secret.md" %}}
+
+{{<imgproc src="/how-tos/access-arm/code-sample.png" resize="700x" class="fill aligncenter" style="width: 400px" declaredimensions=true alt="Code sample page of the CONNECT tab">}}
+
+Then, copy and paste the sample code into a file and run the resulting script to see the resources available on your screen and the position of the end effector output as a [`Pose`](/internals/orientation-vector/).
+For more information, see [`GetEndPosition`](/appendix/apis/components/arm/#getendposition).
+
+{{% /tablestep %}}
+{{% tablestep %}}
+**4. Get the joint positions**
+
+The state of a robot arm can also be described as the **combined positions of each joint** attached to the arm.
+To get that information, add the following code right after the code that gets the end effector pose from the prior code sample:
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+```python {class="line-numbers linkable-line-numbers"}
+# Joint Positions of arm1
+my_arm_joint_positions = await arm_1.get_joint_positions()
+print(f"myArm get_joint_positions return value: {my_arm_joint_positions}")
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+```go {class="line-numbers linkable-line-numbers"}
+// Joint Positions of arm1
+myArmJointPositions, err := arm1.JointPositions(context.Background(), nil)
+if err != nil {
+  logger.Error(err)
+  return
+}
+logger.Infof("myArm JointPositions return value:", myArmJointPositions)
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+Run your code again.
+Each individual value corresponds to the current position of a particular joint on your robot.
+You can also see these values reflected on the **CONTROL** tab in the Viam app for your robot arm.
+
+{{% /tablestep %}}
+{{< /table >}}
+
+## Move the arm
+
+The two main options for specifying arm movement are through **joint position commands** and through **pose commands**.
+
```

Comment:
I'm going to ask Esha and Ray whether we should just take out the pose command stuff from this tutorial anyway since they are deprecated and the motion service does the same thing.

Reply:
@JessamyT can you check what I wrote on line 114? not entirely sure if that's the best description